.detoxrc.js

```diff
@@ -3,7 +3,8 @@ module.exports = {
   testRunner: {
     args: {
       '$0': 'jest',
-      config: 'e2e/jest.config.js'
+      config: 'e2e/jest.config.js',
+      "testTimeout": 100000,
```

Comment:
i believe this may be in milliseconds and causing timeout as too short?
see error on actions https://github.com/zingolabs/zingo-mobile/actions/runs/4691839816/jobs/8317086255?pr=279

```
FAIL e2e/send.test.js (137.045 s)
  Renders wallet data correctly.
    ✕ loads a wallet (3 ms)
    ✕ correctly renders the scanner (1 ms)
    ✕ adds return address to the memo if that option is selected, and correctly renders confirm screen (1 ms)

  ● Renders wallet data correctly. › loads a wallet

    thrown: "Exceeded timeout of 100000 ms for a hook.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

    > 1 | beforeAll(async () => {
        | ^
      2 |   await device.launchApp();
      3 | });
      4 |
```

Reply:
oh! this might be it!

Reply:
well, not really. now it still passes sometimes